docs/user_guide/flyte_fundamentals/registering_workflows.md

```diff
@@ -358,6 +358,34 @@ two GitHub actions that facilitates this:
   of Flyte packages, for example, the `.tgz` archives that are created by
   `pyflyte package`.
 
+### Some CI/CD best practices
+
+Several best practices are essential for ensuring smooth versioning, secure automation and image management. Tools like Docker, Helm and Github Actions help teams streamline their build, tagging, and release processes while maintaining security and scalability. 
```

Comment:
@davidmirror-ops should the tone sound like we're just repeating what Bernhard suggested? i don't think these should come across as if we're suggesting, right?

Reply:
@samhita-alla Agree. 
@RaghavMangla the essence of the issue is to generally capture what was suggested in the conversation. I recognize someone needs significant context to do so. The antigoals are to repeat what Bernhard said, making it sound as a requirement. Almost every organization will have a different way of doing CI/CD. 
Also try to not overemphasize Docker, as Flyte supports any other CRI-compliant container engine.